### PR TITLE
bug(analytics): Remove timestamp property from MediaAutoTrack

### PR DESCRIPTION
**Pull request by stocaaro**
#### Description of changes
Removing timestamp from Personalization properties list as it is set service side and raises an exception when provided by the client.

#### Issue #, if available
https://github.com/aws-amplify/amplify-js/issues/6906

#### Description of how you validated changes
Manual testing.

#### Checklist

- [x] PR description included
- [x] `yarn test` passes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/analytics/src/Providers/AmazonPersonalizeHelper/MediaAutoTrack.ts [hunk 1]
```diff
@@ -173,20 +173,18 @@ export class MediaAutoTrack {
 		const newParams = Object.assign({}, this._params);
 		const { eventData } = newParams;
 		eventData.eventType = eventType;
+		let currentPlayTime;
```
Comment (jimblanc) [inline packages/analytics/src/Providers/AmazonPersonalizeHelper/MediaAutoTrack.ts:176]: Do we want to surface this value in `eventData.properties` somewhere as well so customers can get the value that was formerly in `timestamp`?

**Review by jimblanc — APPROVED** [review]

File: packages/analytics/src/Providers/AmazonPersonalizeHelper/MediaAutoTrack.ts (hunk 1 above)
Comment (elorzafe) [inline packages/analytics/src/Providers/AmazonPersonalizeHelper/MediaAutoTrack.ts:176]: Should this be typed ?

Comment (stocaaro) [reply packages/analytics/src/Providers/AmazonPersonalizeHelper/MediaAutoTrack.ts]: Good call. Updated

**Review by elorzafe — APPROVED** [review]
🚢 

Thanks @stocaaro 🎖️

**Review by jimblanc — APPROVED** [review]

**Pull request merged**